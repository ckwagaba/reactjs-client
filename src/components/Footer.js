import React, { Component } from 'react';

class Footer extends Component {
  render () {
    return (
      <footer>
        {this.props.pageButtons}
      </footer>
    );
  }
}

export default Footer;
