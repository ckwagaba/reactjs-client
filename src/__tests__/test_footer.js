import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/footer';

describe('Footer component', () => {

  const component = <Footer />;

  it('should exist', () => {
    expect(component).toMatchSnapshot();
  });

});
