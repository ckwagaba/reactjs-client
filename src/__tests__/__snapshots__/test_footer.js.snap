// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should exist 1`] = `<Footer />`;
